src/coreclr/src/gc/gcpriv.h

```diff
@@ -1355,7 +1451,7 @@ class gc_heap
                                 gc_heap* hp,
                                 BOOL loh_p);
 
-    PER_HEAP
+    PER_HEAP_ISOLATED
```

Comment:
I didn't see the reason why this needs to be an instance method now instead of static method.

Reply:
not sure I understand...I changed it from an instance method to a static method... the reason is just that I wanted to make all methods that can be static static.

Reply:
Oops, my bad - I read it the wrong way...